pkg/decision/evaluator/condition.go

```diff
@@ -57,6 +67,7 @@ func (c CustomAttributeConditionEvaluator) Evaluate(condition entities.Condition
 	case exactMatchType:
 		matcher = matchers.ExactMatcher{
 			Condition: condition,
+			Logger:    c.logger,
```

Comment:
Why logger not passed in `existsMatchType`

Reply:
No logs were required inside `exists`.